src/harbour-shutter.cpp

```diff
@@ -111,6 +111,9 @@ int main(int argc, char *argv[])
     ResourceHandler handler(app);
     handler.acquire();
 
+    Settings settings(app);
```

Comment:
Can you check if this is needed .. i think it should work ok without these 2 lines